plugins/permission/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@zerodev/permissions",
-    "version": "5.3.1",
+    "version": "5.3.2-alpha-0",
```

Comment:
let's bump the version to `5.4.0` when we release to prod. Since we are adding a new required param.